README.md

```diff
@@ -19,60 +22,69 @@ npm install --save dreamjs
 ## Usage
 
 ### Hello World
+
 ```js
 var dream = require('dreamjs');
 
 var helloworld = dream.output();
 ```
+
 The variable helloworld now contains:
 ```js
 { Dream: 'Hello World' }
 ```
 
 ### Callbacks
-Currently there is two ways you can get the output from DreamJS. By storing it in a variable or by passing a callback to it to receive the result. In the future it will also be possible to use it with Promises and Streams.
+
+Currently there is two ways you can get the output from DreamJS. By storing it in a variable or by passing a callback
```

Comment:
Oh sure.

Let the author (@adleroliveira) looks at my change and in its discretion, will improve the description of your comment.